cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -292,3 +299,51 @@ def test_send_text_custom_report_with_long_condition(
     )
     # assert that the pattern was not found in the logs
     assert log.validate(), "Found error message in the logs."
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1504010])
+@pytest.mark.provider([BaseProvider], selector=ONE_PER_CATEGORY)
```

Comment:
This test collection is too broad, I think, because the fixture being used is deploying a VM. At least `required_flag` should be used, with `provision`, to exclude providers from collection that exclude that test flag.